extensions/flush-dns/package.json

```diff
@@ -16,7 +16,7 @@
   "commands": [
     {
       "name": "index",
-      "title": "Flush DNS",
+      "title": "Flush Dns",
```

Comment:
'Flush Dns' should be 'Flush DNS' since DNS is an acronym

Reply:
Linter was complaining, that why i changed

Reply:
I will look into this, let's use DNS for sure :)